stl/inc/xmemory

```diff
@@ -1079,33 +1191,36 @@ struct _Iterator_base0 {
 // CLASS _Container_proxy
 struct _Container_base12;
 struct _Container_proxy { // store head of iterator chain and back pointer
-    _Container_proxy() noexcept : _Mycont(nullptr), _Myfirstiter(nullptr) {}
-    _Container_proxy(_Container_base12* _Mycont_) noexcept : _Mycont(_Mycont_), _Myfirstiter(nullptr) {}
+    _CONSTEXPR20_DYNALLOC _Container_proxy() noexcept : _Mycont(nullptr), _Myfirstiter(nullptr) {}

```

Comment:
I went back to `_CONSTEXPR20_DYNALLOC` because we still have the bug in MSVC regarding the constexpr destructor which creates a pile of inconsitencies. I would file a transition comment for when this is fixed in the next release?